test/__snapshots__/monitoredQueue.test.ts.snap

```diff
@@ -295,8 +295,7 @@ exports[`MonitoredQueue should create a monitored queue with a lambda SNS listen
             "Arn",
           ],
         },
-        "Runtime": "nodejs18.x",
```

Comment:
This does not make sense.

Will investigate why this is happening 🤔

Reply:
Actually this is a lambda from the LogRetention feature to remove lambda logs after x amount of days